deploy/data_exporter/samconfig.toml

```diff
@@ -0,0 +1,33 @@
+# More information about the configuration file can be found here:
+# https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-config.html
+version = 0.1
+
+[default.global.parameters]
+stack_name = "ynr-data-exporter"
+
+[default.build.parameters]
+cached = true
+parallel = true
+
+[default.validate.parameters]
+lint = true
```

Comment:
What's this doing?

Reply:
No idea! This comes from the `sam init` script that's built in. At a guess, it's running [cfn-lint](https://pypi.org/project/cfn-lint/) on the template at build time.